app/src/main/java/com/sanjeevyadavit/magecart/service/ApiInterface.kt

```diff
@@ -0,0 +1,17 @@
+package com.sanjeevyadavit.magecart.service
+
+import com.sanjeevyadavit.magecart.model.LoginBodyRequest
+import com.sanjeevyadavit.magecart.model.SignupBodyRequest
+import com.sanjeevyadavit.magecart.model.SignupResponse
+import retrofit2.Call
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+interface ApiInterface {
+
+    @POST("V1/integration/customer/token")
+    fun login(@Body loginBodyRequest: LoginBodyRequest): Call<String>
+
+    @POST("V1/customers")
+    fun signup(@Body signupBodyRequest: SignupBodyRequest): Call<SignupResponse>
```

Comment:
You can use coroutines or rxJava for this. Good to learn any one of these. Coroutines is a better option.

Reply:
you mean append `suspend` at the begining, and remove `Call` in return type right?

like this
```
 suspend fun signup(@Body signupBodyRequest: SignupBodyRequest): SignupResponse
```